svf/include/Util/ExtAPI.h

```diff
@@ -224,38 +224,107 @@ class ExtAPI
 
 public:
 
-    class Operation
-    {
-    public:
-        Operation() {};
-
-        Operation(std::string opn, std::vector<std::string> varstr) : op(opn), operandStr(varstr) {};
-
-        std::string getOperator()
-        {
-            return op;
-        }
-
-        std::vector<std::string> getOperandStr()
-        {
-            return operandStr;
-        }
+    enum OperationType {
+        Addr,
+        Copy,
+        Load,
+        Store,
+        Gep,
+        Call,
+        Return,
+        Rb_tree_ops,
+        Memcpy_like,
+        Memset_like
+    };
 
-        std::vector<NodeID> &getOperands()
-        {
-            return operands;
+    class Operand {
+    public:
+        OperationType GetType() const { return opType; }
```

Comment:
make sure all method names start with lowercase.

Add assertions as many as possible for your code.